torchvision/__init__.py

```diff
@@ -95,3 +95,9 @@ def get_video_backend():
 
 def _is_tracing():
     return torch._C._get_tracing_state()
+
+
+def disable_beta_transforms_warning():
+    # Noop, only exists to avoid breaking existing code.
+    # See https://github.com/pytorch/vision/issues/7896
+    pass
```

Comment:
Maybe in the future we'll want to raise a warning here to remove this eventually, but it's probably not worth it